ocs_ci/helpers/dr_helpers.py

```
@@ -1687,71 +1687,163 @@ def validate_vgrc_count():
+            logger.info(
+                f"Verified: lastGroupSyncTime changed to "
+                f"{last_group_sync_time} "
+                f"(sync update {update_num}/{sync_updates_to_wait})"
+            )
+            previous_last_group_sync_time = last_group_sync_time
+
+        # Verify lastGroupSyncTime
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
+        last_group_sync_time_formatted = datetime.strptime(
+            last_group_sync_time, time_format
+        )
```

Ruff (0.15.15)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)